rule/exported.go

```diff
@@ -13,38 +14,96 @@ import (
 	"github.com/mgechev/revive/lint"
 )
 
+// IgnoredWarnings store ignored warnings types
+type IgnoredWarnings struct {
+	Const    bool
+	Function bool
+	Method   bool
+	Type     bool
+	Var      bool
+}
+
+// Name returns the rule name.
```

Comment:
Comment is not related with this function

Reply:
changed comment